### PR TITLE
Improve French translations

### PR DESCRIPTION
**Pull request by jmpp**
I noticed that some translations in French were not really significatives, so I felt free to improve them.

### CODE REVIEW

**Comment by watilde** [comment]
@jmpp Oh thank you! Do you know someone who can review this...?

**Comment by jmpp** [comment]
> @jmpp Oh thank you! Do you know someone who can review this...?

(I have missed your answer. Sorry about this)
Maybe @isaacs or @tdd could review this?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "how-to-npm",
-  "version": "2.5.1",
+  "name": "@jmpp/how-to-npm",
```
Comment (juanpicado) [inline package.json]: 🤔🤔🤔🤔🤔🤔🤔🤔

**Review by juanpicado — CHANGES_REQUESTED** [review]
You have changes the package name, why? It is a mistake?

File: package.json (hunk 1 above)
Comment (jmpp) [reply package.json]: My bad! That's a wrong manipulation from yesterday.

**Explanations:**
I wanted to get an updated version with improved French translations from Dec 4th 2018, that still wasn't merged.

I took back my old repo, made some changes and updated the package's name in order to publish a scoped package : https://www.npmjs.com/package/@jmpp/how-to-npm for my students.

Working in a rush, I forgot to create another branch than `master` for this, so my "personal" updates came into this pull-request that was still waiting ...

As it's now not relevant anymore, I've hard-removed theses commits from `master`

Comment (juanpicado) [reply package.json]: Np thanks for providing details 🙌

Comment (jmpp) [reply package.json]: @juanpicado So, I did remove this original pull request that came with my "accidental" other commits.

But would you be interested in another clean PR with the original proposal of the improved French translations? :)

**Pull request closed without merging**